install.bash

```diff
@@ -29,22 +29,18 @@ install() {
         echo "Unexpected mode: $1"
     fi
 
-    echo "# allow pem command" >> ~/.bash_profile
-    echo "export PATH=$PEM_INSTALL_PATH/bin"':$PATH' >> ~/.bash_profile
-    echo "# allow pem completion" >> ~/.bash_profile
-    echo "[ -f $PEM_INSTALL_PATH/bin/pem-completion ] && . $PEM_INSTALL_PATH/bin/pem-completion" >> ~/.bash_profile
-    echo ============== ~/.bash_profile ================
-    grep pem ~/.bash_profile
-    echo ===============================================
+    echo "Add the following to your .bash_profile to allow:"
+    echo "  pem command"
+    echo "    export PATH=$PEM_INSTALL_PATH/bin:\$PATH"
+    echo "  pem completion"
+    echo "    [ -f $PEM_INSTALL_PATH/bin/pem-completion ] && . $PEM_INSTALL_PATH/bin/pem-completion"
```

Comment:
@AntonKarabaza, the best way to add this instructions is to copy and paste. But in current implementation I can't do that:
1. Please add # before comments
2. Please make it in copy/paste format (remove extra spaces in the begging go the line